modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-requirements.adoc

```diff
@@ -0,0 +1,22 @@
+[[requirements]]
+== Requirements
+
+To run {pluginname} On-Premises a Docker environment is required. Alternatively, use CaaS available from your cloud provider, like AWS ECS, Google GKE, or Azure ACS.
+
+There are many factors affecting {pluginname} On-Premises performance. The most influential are the size of exported content, the size of images, and the number of concurrent requests. Also, because your application can prioritize fast response times, or it should handle high load, it is impossible to provide one recommended server specification, that will fit all use cases.
+
+Assuming response time below 10 seconds, one server (2CPU 2GB RAM) with 1 docker container can handle:
+
+* up to 40 concurrent requests with an average content of 1 A4 page (~1k characters and 1 image)
+* up to 25 concurrent requests with an average content of 5 A4 pages (~7,5k characters and 5 images)
+* up to 10 concurrent requests with an average content of 20 A4 pages (~30k characters and 20 images)
+
+> The above concurrent requests numbers are not a hard limit of {pluginname} On-Premises instance. It can handle more concurrent requests, but the response time will be longer.
```

Comment:
Convert into a note-box

Reply:
```suggestion
[NOTE]
The above concurrent requests numbers are not a hard limit of {pluginname} On-Premises instance. It can handle more concurrent requests, but the response time will be longer.
```